compiler/src/option/GlobalOptions.cpp

```diff
@@ -42,6 +42,26 @@ EnumOpt spnc::option::vectorLibrary{"vector-library", NONE,
 
 Option<bool> spnc::option::replaceGatherWithShuffle{"use-shuffle", false};
 
+Option<unsigned>spnc::option::slpMaxAttempts{"slp-max-attempts", 5, {depends(spnc::option::cpuVectorize, true)}};
```

Comment:
I don't think it's necessary to perform additional experiments. But if your experiments were successful with `1`, then let's make this the default here to avoid excessive runtime for unsuspecting users.

Reply:
Changed in 3f65e782bd518e9678d67bde32f8b80b819a952f.